polyfills/ElementTraversal/polyfill.js

```diff
@@ -0,0 +1,43 @@
+(function(){
```

Comment:
If you are importing a 3rd party polyfill using the config `install` mechanism, you don't need to commit the original source, it's imported automatically at build time.